styles/styles.css

```diff
@@ -779,3 +844,356 @@ main .section.leadership-content .columns > div > div > h6 {
 }
 
 
+main .zelta-section {
+  padding-bottom: 0;
+}
+
+main .zelta-section::after {
```

Comment:
class name should not be this tied to the content in this case

Reply:
In this case its a border line basically so "colorful-border" or "gradient-border" might be more appropriate